marshmallow_utils/schemas/identifier.py

```diff
@@ -83,7 +84,10 @@ def validate_identifier(self, data, **kwargs):
                 raise ValidationError("Missing required scheme.")
 
             # Check if identifier is valid according to scheme.
-            if scheme not in detected_schemes:
+            # NOTE: This is required to pass PEP-8
+            condition = scheme not in detected_schemes and \
```

Comment:
I see the problem... I do not have any other idea on how to solve it. Any idea @ntarocco. The issue is PEP-8:

Too long, more than 79
```python
if scheme not in detected_schemes and not self.unknown_schemas_accepted:
```

Complains about the following `raise` being at the same indentation level than the condition
```python
if scheme not in detected_schemes and \
    not self.unknown_schemas_accepted:
```

Any other try of spaces (or no) indentation complains about too much or too few indentation
```python
if scheme not in detected_schemes and \
 not self.unknown_schemas_accepted:
```

The only other option is to change the name, to something shorter. But I do like the fact that is self-explanatory as it is.

Reply:
That's why I simply use [Black](https://github.com/psf/black): I don't want to spend a single second with this! :)
Another way is to use PyCharm, it helps. It looks to me that double indentation works:

![Screenshot 2021-04-14 at 14 15 30](https://user-images.githubusercontent.com/2089455/114708681-f553da80-9d2b-11eb-9dfb-8f2441a50440.png)

While wrong indentation it will complain:

![Screenshot 2021-04-14 at 14 16 20](https://user-images.githubusercontent.com/2089455/114708727-07357d80-9d2c-11eb-876f-a8b8d682e8ce.png)